src/Middleware/ErrorResponseGenerator.php

```diff
@@ -103,11 +103,10 @@ private function prepareTemplatedResponse($e, ServerRequestInterface $request, R
 
     /**
      * @param \Throwable|\Exception $e
-     * @param ServerRequestInterface $request
      * @param ResponseInterface $response
      * @return ResponseInterface
      */
-    private function prepareDefaultResponse($e, ServerRequestInterface $request, ResponseInterface $response)
```

Comment:
I did this originally for symmetry; all `prepare*()` methods received the same arguments. However, being marked `private`, there's no reason to include the argument if it's not being used.